### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by PingalPie** [ncmpcpp/ncmpcpp#540]
**Describe the bug**
I was starting ncmpcpp but it gave me error

**Screenshots/Logs**
[Screenshot](http://0x0.st/oT5F.png)

**Desktop (please complete the following information):**
 - OS: Arch linux 
 - Terminal: suckless' st 
 - Shell: zsh

**Additional context**

**Comment by Spacefreak18** [comment]
I don't think this is a bug. I think your package manage either installed that library in the wrong location, not at all, wrong name, etc. 

Or you accidentally deleted it.  

You should probably reach out to the Arch Linux community for bug reports / assistance.

**Comment by leoperegrino** [comment]
This is likely due to updates with the shared objects, which are dependencies dynamically linked to your ncmpcpp binary. If you compiled from source, it will be enough to recompile and install it again. If you just used the package manager, you may need to check if the release is compatible with the shared objects. Either way, compiling and installing will probably work.
